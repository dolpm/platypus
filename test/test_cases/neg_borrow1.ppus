/* try to immutably borrow a mutably borrowed variable */
pipe main |> [] |> [] |> unit {
  int x <| 0;
  {
    ~&int y <| ~&y;
    &int z <| &x;
  }
  |> ();
}