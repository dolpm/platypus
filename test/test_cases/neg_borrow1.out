pipe main |> [] |> [] |> unit {
  (unit : );
{
  (unit : );
  (unit : );
}
  (unit : );
}
